frontend/components/organisms/CourseDetailVideoSection.tsx

```diff
@@ -69,6 +74,20 @@ export function CourseDetailVideoSection({
   isProfileOpen?: boolean
   closeProfileModal?: () => void
 }) {
+
+// 動画のURL、タイトル取得
+const shareUrl = location.href;
+const shareTitle = selectedVideo?.sections.videos.name;
+
+// 動画のURLをクリップボードにコピー
+const { showSuccessToast } = useCustomToast()
+function copyUrlToClipboard(shareUrl){
+  navigator.clipboard.writeText(shareUrl)
+  .then(function(){
+    showSuccessToast('動画のURLをクリップボードにコピーしました!')
+  }
+  )}
```

Comment:
基本的にメソッドを定義するときはアロー関数型で書きましょう！
https://qiita.com/may88seiji/items/4a49c7c78b55d75d693b

```suggestion
  const copyUrlToClipboard = (shareUrl: string) => {
    navigator.clipboard.writeText(shareUrl).then(
      function () {
        showSuccessToast('動画のURLをクリップボードにコピーしました!')
      },
      function () {
        showErrorToast('動画のURLのコピーに失敗しました')
      },
    )
  }
```